packages/components/__tests__/reactIntlMock.ts

```diff
@@ -0,0 +1,37 @@
+import { MessageDescriptor } from "@cultureamp/i18n-react-intl"
+import { vi } from "vitest"
+
+const replaceInputValue = (str: string, value: string): string => {
+  const regex = /{([^}]+)}/g
+  return str.replace(regex, (match, capturedValue) => {
+    if (capturedValue) {
+      return value[capturedValue]
+    }
+    // Handle other captured values here if needed
+    return match
+  })
+}
+
+export const reactIntlMock = (): any => {
```

Comment:
@HeartSquared I resolved the react intl problem with a global mock, but did you try this before doing the custom render? Was there a reason why this didn't work for Jest?